arangod/Statistics/ConnectionStatistics.cpp

```diff
@@ -33,13 +38,50 @@ using namespace arangodb;
 // --SECTION--                                                  global variables
 // -----------------------------------------------------------------------------
 
-static size_t const QUEUE_SIZE = 64 * 1024 - 2;  // current (1.62) boost maximum
+namespace {
+// initial amount of empty statistics items to be created in statisticsItems
+constexpr size_t kInitialQueueSize = 32;
```

Comment:
maybe this can be static?
```suggestion
static constexpr size_t kInitialQueueSize = 32;
```

Reply:
It doesn't need to be static, because it is already in the anonymous namespace of the translation unit.